ospf6d/ospf6_interface.h

```diff
@@ -24,13 +24,43 @@
 #include "qobj.h"
 #include "hook.h"
 #include "if.h"
+#include "ospf6d.h"
 
 /* Debug option */
 extern unsigned char conf_debug_ospf6_interface;
 #define OSPF6_DEBUG_INTERFACE_ON() (conf_debug_ospf6_interface = 1)
 #define OSPF6_DEBUG_INTERFACE_OFF() (conf_debug_ospf6_interface = 0)
 #define IS_OSPF6_DEBUG_INTERFACE (conf_debug_ospf6_interface)
 
+struct ospf6_auth_cfg {
+	/* higher order Sequence Number */
+	uint32_t seqnum_h;
```

Comment:
This seems to be a mixture of config, state and statistics.  That's OK but please change the name from `ospf6_auth_cfg` to something that does not include "cfg" (`ospf_auth_data` maybe?), and add notes in the comments which is config, which is state, and which is statistics.

Reply:
Done

Reply:
It's not done; what I asked was to make clear what is **config**, what is **operational state**, and what is **counters/statistics**.  A `/* hash algorithm type */` comment on a `hash_algo` member is pointless.  A `/* configured */` comment is not.